server/routes/officeRoute.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+import OfficeController from '../controllers/OfficeController';
+import OfficeValidator from '../middlewares/OfficeValidator';
+
+const router = express.Router();
+
+
+router.post(
+  '/offices',
+  OfficeValidator.validateFields,
+  OfficeController.postOffice,
+);
+
+router.get(
+  '/offices/:id',
+  OfficeController.getOffice,
```

Comment:
You're not validating that I actually passed in an ID. I think you should do that.